### PR TITLE
fix C-style deprecations in iasm.d and iasm64.d

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: test/runnable/iasm64.d [hunk 1]
```diff
@@ -6401,7 +6401,7 @@ L1:     pop     RAX;
 
     foreach (ref i, b; data)
     {
-        // printf("data[%d] = 0x%02x, should be 0x%02x\n", i, p[i], b);
+        printf("data[%d] = 0x%02x, should be 0x%02x\n", i, p[i], b);
```
Comment (ZombineDev) [inline test/runnable/iasm64.d]: This looks like a debugging leftover; it seams it should be commented-out.

**Review by PetarKirov — APPROVED** [review]
Apart from the comment below, LGTM.

**Comment by schuetzm** [comment]
s/deprecations/declarations/ in the commit subject

**Comment by WalterBright** [comment]
> s/deprecations/declarations/ in the commit subject

I meant deprecations, because without this PR one gets a blizzard of C style deprecation messages.

**Comment by andralex** [comment]
noice thx

**Pull request merged**